src/actions/typeGenerator.js

```diff
@@ -8,6 +8,9 @@ const typeGenerator = name => ({
   loading: `${name}_LOADING`,
   success: `${name}_SUCCESS`,
   failure: `${name}_FAILURE`,
+  error: `${name}_ERROR`,
```

Comment:
#### Nitpick
- Not sure why this is necessary.

Reply:
I am using it in many other places to populate the store with different types of errors.